LIMS_IMAGE/web/src/urls.py

```diff
@@ -28,6 +28,7 @@
     path('orders/', include('orders.urls')),
     path('laboratory/', include('laboratory.urls')),
     path('', TemplateView.as_view(template_name='home.html'), name='home'),  # new
+    path('', TemplateView.as_view(template_name='shopping.html'), name='order_form'),  # new
```

Comment:
this isn't needed. The shopping url, is already added in from orders.urls which is included on line 28

Reply:
I wanted for the appendix to go back to order form with the back bottom, but it is going back to home anyway. So I will remove it.